internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr_test.go

```diff
@@ -84,6 +91,9 @@ func TestKafkaManager_Reconcile(t *testing.T) {
 					DeprovisionExpiredKafkasFunc: func() *errors.ServiceError {
 						return errors.GeneralError("failed to deprovision expired kafkas")
 					},
+					ListAllFunc: func() (dbapi.KafkaList, *errors.ServiceError) {
+						return dbapi.KafkaList{}, nil
+					},
 				},
 				clusterService: &services.ClusterServiceMock{
 					FindStreamingUnitCountByClusterAndInstanceTypeFunc: func() (services.KafkaStreamingUnitCountPerClusterList, error) {
```

Comment:
Might be worth adding another test case here to ensure that if the newly added ListAll call fails, Reconcile() should return that error.